admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -2131,58 +2134,165 @@ protected function generateDeletionPreview($event_ids)
         );
     }
 
+    /**
+     * Checks for a POST submission
+     * @since $VID:$
+     */
+    protected function confirmDeletion()
+    {
+        $deletion_job_code = isset($this->_req_data['deletion_job_code']) ? $this->_req_data['deletion_job_code'] : '';
```

Comment:
👽 

```suggestion
        $deletion_job_code = isset($this->_req_data['deletion_job_code']) ? sanitize_key( $this->_req_data['deletion_job_code'] ) : '';
```

Reply:
thanks! yeah somehow I totally overlooked all sanitizing :disappointed_relieved:

Reply:
applying to new PR